src/index.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+
+const path = require("path");
+
+const routes = require("./routes");
+
+const PORT = process.env.PORT || 4000;
+
+const app = express();
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(express.static(path.join(__dirname, "../public")));
+app.use(routes);
+
+// TODO check return for both app.get and /vs*
+app.get("/", (req, res) => {
```

Comment:
These two routes here will never be reached as you are routing all requests to the `routes` folder on line 17. I'm not sure why you've left them in here?